tm2/pkg/std/memfile.go

```diff
@@ -37,14 +38,32 @@ func (mempkg *MemPackage) IsEmpty() bool {
 	return len(mempkg.Files) == 0
 }
 
-const rePathPart = `[a-z][a-z0-9_]*`
-
 var (
-	rePkgName      = regexp.MustCompile(`^[a-z][a-z0-9_]*$`)
-	rePkgOrRlmPath = regexp.MustCompile(`gno\.land/(?:p|r)(?:/` + rePathPart + `)+`)
-	reFileName     = regexp.MustCompile(`^([a-zA-Z0-9_]*\.[a-z0-9_\.]*|LICENSE|README)$`)
+	rePathPart = regexp.MustCompile(`^[a-z0-9_]+$`)
+	rePkgName  = regexp.MustCompile(`^[a-z][a-z0-9_]*$`)
+	reFileName = regexp.MustCompile(`^([a-zA-Z0-9_]*\.[a-z0-9_\.]*|LICENSE|README)$`)
```

Comment:
What do you think about adding a section in the documentation, in this PR, on what are valid package / realm names?

Reply:
@leohhhn what is best place to put this?

Reply:
I suggest you create a page under the [Concepts section](https://docs.gno.land/concepts/), titled "Namespaces". I would structure the page this way:
- Short intro on Gno paths, what they are, why they exist, how they compare/work with gno addresses (address is a path derivative). Possibly compare with other blockchain systems
- Namespaces
    - Content from this PR, explaining to people what are the currently allowed paths
    - Mention plans for the future, based on [this](https://github.com/gnolang/gno/issues/1107)

Reply:
Thanks @leohhhn. I'll work on this here -> https://github.com/gnolang/gno/pull/1909